<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionControlPane</class>
 <widget class="QWidget" name="VersionControlPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionRefresh"/>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="commitListView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="toolBar2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionBackwards"/>
     <addaction name="actionCompare_Commits"/>
     <addaction name="separator"/>
     <addaction name="actionUpload"/>
     <addaction name="actionPack_Zip"/>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="diffListView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Refresh_16x.svg</normaloff>:/Resource/icons/Refresh_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBackwards">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Backwards_16x.svg</normaloff>:/Resource/icons/Backwards_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Backwards</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/BatchCheckIn_16x.svg</normaloff>:/Resource/icons/BatchCheckIn_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionPack_Zip">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ZipFile_16x.svg</normaloff>:/Resource/icons/ZipFile_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Pack Zip</string>
   </property>
  </action>
  <action name="actionCompare_Commits">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CompareFolders_16x.svg</normaloff>:/Resource/icons/CompareFolders_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Compare Commits</string>
   </property>
  </action>
  <action name="actionMark_as_Dev">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/flag_dev.svg</normaloff>:/Resource/icons/flag_dev.svg</iconset>
   </property>
   <property name="text">
    <string>Mark as Dev</string>
   </property>
  </action>
  <action name="actionMark_as_Prod">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/flag_prod.svg</normaloff>:/Resource/icons/flag_prod.svg</iconset>
   </property>
   <property name="text">
    <string>Mark as Prod</string>
   </property>
  </action>
  <action name="actionMark_as_Other">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/flag_other.svg</normaloff>:/Resource/icons/flag_other.svg</iconset>
   </property>
   <property name="text">
    <string>Mark as Other</string>
   </property>
  </action>
  <action name="actionClear_Flags">
   <property name="text">
    <string>Clear Flags</string>
   </property>
  </action>
  <action name="actionClear_All_Flags">
   <property name="text">
    <string>Clear All Flags</string>
   </property>
  </action>
  <action name="actionSynchronous_Deletion">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/DeleteTranslation_16x.svg</normaloff>:/Resource/icons/DeleteTranslation_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Synchronous Deletion</string>
   </property>
  </action>
  <action name="actionPack_Selection_To_Zip">
   <property name="text">
    <string>Pack Selection To Zip</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/OpenFile_16x.svg</normaloff>:/Resource/icons/OpenFile_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/OpenFolder_16x.svg</normaloff>:/Resource/icons/OpenFolder_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionCopy_Path">
   <property name="text">
    <string>Copy Path</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
